keylime/tenant.py

```diff
@@ -1118,7 +1092,6 @@ def do_quote(self):
                 raise UserError(
                     f"Posting of Encrypted U to the Cloud Agent failed with response code {response.status_code} ({response.text})")
         except Exception as e:
```

Comment:
The try-catch block can be fully removed, instead of capturing and re-raising the exception.

Reply:
Thanks for the review, I'll remove it.